### ISSUE TITLE
A11y: The active entry of the main navigation cannot be determined programmatically

### ISSUE THREAD

**Issue by 73nici** [shopware/platform#7088]
*PHP Version:* 8.2   | *Shopware Version:* 6.6.9.0  | *Affected area / extension:* Platform(Default) 

 ----

*_Actual behaviour:_*

The active entry in the flyover of the main navigation (CSS selector .nav-link.active) is only highlighted visually and cannot be determined programmatically. This excludes e.g. screen reader users from the announcement of this status.

*_Expected behaviour:_*

An aria-current=‘page’ attribute must be added to the a HTML element of the active entry.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current#page

*_How to reproduce:_*

Enable `ACCESSIBILITY_TWEAKS` Feature-Flag:
- visit a page which shows the main navigation

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-40842**



[created from NEXT-40842, comment 541807]

**Comment by shopwareBot** [comment]
[public] 

 > Priority changed to: **High**

 Thank you for your report. Based on the information in this report, we have decided to investigate this issue in more detail and are planning to provide a fix for this issue. Although we are always eager to solve such problems as soon as we can, we can't give you a more specific time frame right now. We appreciate your patience and thank you for your understanding.



[created from NEXT-40842, comment 542002]
